src/Tizen.NUI.Components/Controls/ItemsView/ItemSource/IItemSource.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+
+namespace Tizen.NUI.Components
+{
+	/// <summary>
+    /// Base interface for encapsulated data source in ItemsView.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+	public interface IItemSource : IDisposable
```

Comment:
`[EditorBrowsable(EditorBrowsableState.Never)]`가 빠졌습니다.
```suggestion
	[EditorBrowsable(EditorBrowsableState.Never)]
	public interface IItemSource : IDisposable
```